docs/hub/spaces-oauth.md

```diff
@@ -60,12 +61,18 @@ You can add a custom relative redirect path by setting `hf_oauth_redirect_path`
 
 ## Scopes
 
-The following scopes are available:
+The following scopes are always included for Spaces:
 
 - `openid`: Get the ID token in addition to the access token.
 - `profile`: Get the user's profile information (username, avatar, etc.)
 
-You should use `"openid profile"` as the scope for your OAuth app.
+Those scopes are optional and can be added by setting `hf_oauth_scopes` in your Space's metadata:
```

Comment:
```suggestion
Those scopes are optional and can be added by setting `hf_oauth_scopes` in your Space's metadata (as a space-separated string):
```

Reply:
then the example at https://github.com/huggingface/hub-docs/pull/1134/files#diff-8c77839fc5d40b69830303ec9567fbc0a4fbbefabf3309abef7de83f758f2652R37 is wrong, no?